libraries/AP_VideoTX/AP_SmartAudio.cpp

```diff
@@ -228,10 +228,18 @@ void AP_SmartAudio::update_vtx_params()
                 break;
             default:    // v1
                 switch(vtx.get_configured_power_level()) {
-                    case 1: set_power(16); break; // 200mw
```

Comment:
I'd leave this change out.  It doesn't make any functional difference and the author probably preferred it as it was.